packages/modules/link-modules/src/definitions/order-fulfillment.ts

```diff
@@ -0,0 +1,62 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+import { LINKS } from "@medusajs/utils"
+
+export const OrderFulfillment: ModuleJoinerConfig = {
+  serviceName: LINKS.OrderFulfillment,
+  isLink: true,
+  databaseConfig: {
+    tableName: "order_fulfillment",
+    idPrefix: "ordful",
+  },
+  alias: [
+    {
+      name: ["order_fulfillment", "order_fulfillments"],
+      args: {
+        entity: "LinkOrderFulfillment",
+      },
+    },
+  ],
+  primaryKeys: ["id", "order_id", "fulfillment_id"],
+  relationships: [
+    {
+      serviceName: Modules.ORDER,
+      primaryKey: "id",
+      foreignKey: "order_id",
+      alias: "order",
+    },
+    {
+      serviceName: Modules.FULFILLMENT,
+      primaryKey: "id",
+      foreignKey: "fulfillment_id",
+      alias: "fulfillment",
+      args: {
+        methodSuffix: "Fulfillments",
+      },
+      isList: true,
```

Comment:
question: Shouldn't this be in the `extends`?

cc @carlos-r-l-rodrigues

Reply:
Not sure if this was already changed. But the file looks ok.